scalableminds/webknossos
# Show email verification status in users table and make it filterable

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- create new account (automatically not activated and not verified)
- go to users list and check out the status column (tooltips and filters should make sense)
![image](https://github.com/scalableminds/webknossos/assets/2486553/b6fcfc10-7189-47e3-a4c4-082308622412)
![image](https://github.com/scalableminds/webknossos/assets/2486553/54fa2718-ab4f-4688-a547-f5fedc4e5b36)

### Issues:
- fixes #7228
